### DISCUSSION TITLE
Idea : Form - Password - CapsLock warning

### DISCUSSION THREAD

**Discussion by MGeurts** [tasteui/tasteui#446] [Ideas]
![GMKuYByXgAAwkI7](https://github.com/tallstackui/tallstackui/assets/5404867/a1b21257-5a82-48de-8105-05d6edc9541d)

**Comment by devajmeireles** [comment]
Done! Released today in our new release: v1.26.0: https://tallstackui.com/docs/form/password#caps-lock-indicator

**Comment by MGeurts** [comment]
Very nice. Thank you.
